RIOT-OS/RIOT
# TI CC3200 SimpleLink RIOT support

Hello dear RIOT-Devs,
hope this port is of some value too you and I would really appreciate any feedback.

### Contribution description
I have added basic support for the cc3200-launchxl (including SPI, UART, GPIO and partially timers). The board can be flashed and debugged on linux as well as macOS (debugging on macOS has some problems).

### Testing procedure

Debugging should work right out of the box if OpenOCD 0.10 and arm-none-eabi-gdb  are installed. Therefore this command should work:  
`make all debug`
This will halt the board and set a breakpoint at main allowing for debugging via GDB. 

Flashing is more complicated since it requires a proprietary TI tool.  Shortly it is required to download the cc3200prog tool via Energia (or direct link) and set the path to the containing folder via ENERGIA_TOOL env variable when flashing the board. A more in depth instruction on that can be found at boards/cc3200-launchxl/doc.txt


### Issues/PRs references
Fixes #9385
